src/krati_Shukla/ReturningValues/SumAverage.java

```diff
@@ -0,0 +1,89 @@
+/*1) Method sum() - non static method, accept 3 parameters of type double & return answer
+
+2) Method average()- non static method, accept 3 parameters of type double & return answer
+
+3) Method isAnswerAboveExpecation() - non static method, take one parameter (answer which you received from average method) 
+and return true if value is more then 50 else false.
+
+4) Method isEligible() - static method,take one parameter (answer which you received from average method) return true if value is more then 50 
+or divisible by 2. Eligibility will be printed in main method. 
+
+5) Method getYourGrade() - static method, take one parameter (answer which you received from average method) 
+return "A Grade" if average is more then 80 else "B grade", main method will print grade received from getYourGrade() method.
+
+2) Method average()- non static method, accept 3 parameters of type double & return answer
+
+3) Method isAnswerAboveExpecation() - non static method, take one parameter (answer which you received from average method) 
+and return true if value is more then 50 else false.
+
+4) Method isEligible() - static method,take one parameter (answer which you received from average method) return true if value is more then 50 
+or divisible by 2. Eligibility will be printed in main method. 
+
+5) Method getYourGrade() - static method, take one parameter (answer which you received from average method) 
+return "A Grade" if average is more then 80 else "B grade", main method will print grade received from getYourGrade() method.*/
+
+package krati_Shukla.ReturningValues;
+
+public class SumAverage {
+	
+	double totalSum (double num1, double num2, double num3) {
+		double sum = num1+num2+num3;
+		return sum;
+	}
+	
+	double average (double num1, double num2, double num3) {
+		double sum = totalSum(num1, num2, num3);
+		double avg = (sum)/3;
+		return avg;
+	}
+	
+	boolean isAnswerAboveExpecation(double avg) {
+		if (avg>50)
+			return true;
+		else
+			return false;
+	}
+	
+	static boolean isEligible(double avg) {
+		SumAverage sumAverage1 = new SumAverage();
+		boolean eligible = sumAverage1.isAnswerAboveExpecation(avg);
+		if ((eligible == true) || (avg%2==0))
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.